test/e2e_node/gci-init.sh

```diff
@@ -26,7 +26,8 @@ fi
 
 if [ "${CONTAINERD_CGROUPV2:-"false"}"  == "true" ]; then
   # check cos image
-  if uname -a | grep -q cos; then
+	OS_ID=$(cat /etc/os-release | grep "^ID=.*" | cut -c 4-)
```

Comment:
How about `grep '^ID=cos-'`? since we use the ID only for cos.

https://www.freedesktop.org/software/systemd/man/os-release.html

https://cloud.google.com/container-optimized-os/docs/concepts/versioning

Reply:
Or just source the file and echo `$ID`;

```bash
if [ -r /etc/os-release ]; then
		OS_ID="$(. /etc/os-release && echo "$ID")"
fi
```

Reply:
Thanks, update the line with suggested ways using source the file and echo $ID